### ISSUE TITLE
Test failures with Bedtools 2.26.0

### ISSUE THREAD

**Issue by rekado** [daler/pybedtools#261]
Since there are 16+ test failures when building 0.7.10 with Bedtools 2.27.1, I tried again with Bedtools 2.26.0 and got only 2 failures:

```
======================================================================
FAIL: pybedtools.test.test1.test_to_dataframe
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/gnu/store/am211hm7jb2y0schyzz4bx68z1i19l0h-python-nose-1.3.7/lib/python3.6/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/tmp/guix-build-python-pybedtools-0.7.10.drv-0/pybedtools-0.7.10/pybedtools/test/test1.py", line 1656, in test_to_dataframe
    assert results == expected
AssertionError: 
>>  assert '  seqname source           ...           frame              attributes\n0    chr1   fake           ...               .                ID=gene1\n1    chr1   fake           ...               .  ID=mRNA1;Parent=gene1;\n2    chr1   fake           ...               .   ID=CDS1;Parent=mRNA1;\n3    chr1   fake           ...               .   ID=CDS2;Parent=mRNA1;\n4    chr1   fake           ...               .               ID=rRNA1;\n\n[5 rows x 9 columns]' == '  seqname source feature  start   end score strand frame  \\\n0    chr1   fake    gene     50   300     .      +     .   \n1    chr1   fake    mRNA     50   300     .      +     .   \n2    chr1   fake     CDS     75   150     .      +     .   \n3    chr1   fake     CDS    200   275     .      +     .   \n4    chr1   fake    rRNA   1200  1275     .      +     .   \n\n               attributes  \n0                ID=gene1  \n1  ID=mRNA1;Parent=gene1;  \n2   ID=CDS1;Parent=mRNA1;  \n3   ID=CDS2;Parent=mRNA1;  \n4               ID=rRNA1;  '
    

======================================================================
FAIL: pybedtools.test.test1.test_issue_157
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/gnu/store/am211hm7jb2y0schyzz4bx68z1i19l0h-python-nose-1.3.7/lib/python3.6/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/tmp/guix-build-python-pybedtools-0.7.10.drv-0/pybedtools-0.7.10/pybedtools/test/test1.py", line 2029, in test_issue_157
    assert non_dataframe == from_dataframe
AssertionError: 
    '' = str(<BedTool(/tmp/guix-build-python-pybedtools-0.7.10.drv-0/pybedtools-0.7.10/pybedtools/test/tmp/pybedtools.ux5e1rhl.tmp)>.intersect(<BedTool(/tmp/guix-build-python-pybedtools-0.7.10.drv-0/pybedtools-0.7.10/pybedtools/test/tmp/pybedtools.pff0phk2.tmp)>))
>>  assert '20\t14370\trs6054257\tG\tA\t29\tPASS\tNS=3;DP=14;AF=0.5;DB;H2\tGT:GQ:DP:HQ\t0|0:48:1:51,51\t1|0:48:8:51,51\t1/1:43:5:.,.\n' == ''
```

For completeness I should add that I disabled `pybedtools.test.test_helpers.test_chromsizes` (as there is no internet connection while the tests run) and `pybedtools.test.test_scripts.test_venn_mpl` (as there's no GUI).

**Comment by daler** [comment]
Thanks @rekado, there were changes in 2.27.1 that affected random shuffling. I'm updating all the tests to reflect those new outputs, and modernizing tests across the board.

**Comment by daler** [comment]
Another note: these tests are dependent on the particular version of pandas installed, as the way dataframes are printed varies by pandas version.

Anyway, hopefully this one and #260 are fixed by PR #264.
